eng/pipelines/corefx-base.yml

```diff
@@ -79,9 +79,6 @@ jobs:
         - ${{ if eq(job.submitToHelix, 'true') }}:
           - group: DotNet-HelixApi-Access
 
-          - ${{ if eq(parameters.isOfficialBuild, 'false') }}:
```

Comment:
Another option, rather than explicitly specifying the values for "isOfficialBuild" or not like this...

```
- ${{ if eq(parameters.isOfficialBuild, 'false') }}:
  - _officialBuildArgument: ''
- ${{ if eq(parameters.isOfficialBuild, 'true') }}:
  - _officialBuildArgument: /p:somethingofficial=badge

- ${{ if eq(parameters.isOfficialBuild, 'false') }}:
  - _anotherOfficialBuildArgument: ''
- ${{ if eq(parameters.isOfficialBuild, 'true') }}:
  - _anotherOfficialBuildArgument: /p:somethingelseofficial=pie
```

...is to just specify the defaults and then override for official builds like this...

```
variables:
- _officialBuildArgument: ''
- _anotherOfficialBuildArgument: ''
  
- ${{ if eq(parameters.isOfficialBuild, 'true') }}:
  - _officialBuildArgument: /p:somethingofficial=badge

- ${{ if eq(parameters.isOfficialBuild, 'true') }}:
  - _anotherOfficialBuildArgument: /p:somethingelseofficial=pie
```

Reply:
I didn't know this was possible, last time I tries that, I got: variable is already defined (stating you can't have two declarations of the same key in yaml).